src/hb-subset-cff-common.hh

```diff
@@ -133,13 +133,56 @@ struct str_encoder_t
       (void) hb_uselocale (((void) freelocale (clocale), oldlocale));
 
       char *s = buf;
+      size_t len;
+      char *comma = strchr (s, ',');
+      if (comma)
+	*comma = '.';
```

Comment:
Is this because of locales? That shouldn't happen. What's this for?

Reply:
A preexisting comment later in the function says “Comma for some European locales in case no uselocale available”, so I assume it can happen. I added this line so that rewrite rules that check for the decimal point would work in that situation.

Reply:
Ughh. Okay.